public/app/components/user/editUser/fieldTemplates/editUserFieldTemplates.dir.js

```diff
@@ -42,11 +42,19 @@
 	}
 	function editBirthdayField() {
 		return {
-			template: ['<form>',
-			      	   '<md-input-container>',
-          			   '<label>Birthday</label>',
-					   '<md-datepicker ng-model="profile.date_of_birth" md-placeholder="MM/DD/YYYY" flex></md-datepicker>',
-					   '</md-input-container>',
+			template: ['<form layout="row"  ng-controller="birthdayController as birthday" flex>',
+			           '<md-input-container>',
+          			   '<label>Month</label>',
+					   '<md-select ng-model="profile.date_of_birth.month" ng-change="selectMonth(profile.date_of_birth.month,profile.date_of_birth.year)"><md-option ng-repeat="month in months " placeholder="Month" ng-value="month">{{month.name}}</md-option></md-select>',
```

Comment:
`profile.date_of_birth.month, profile.date_of_birth.year`

watch out for your spaces after `,`